operations/deploy/tests/test_operations.py

```diff
@@ -0,0 +1,141 @@
+import os
+import sys
+import uuid
+import tempfile
+from unittest import mock
+
+from dffml.df.types import Input, DataFlow
+from dffml.df.base import opimp_in
+from dffml.df.memory import MemoryOrchestrator
+from dffml.util.asynctestcase import AsyncTestCase
+
+from dffml.base import BaseDataFlowFacilitatorObjectContext
+
+
+from dffml.operation.output import GetSingle
+from deploy.operations import *
+from dffml_feature_git.feature.operations import clone_git_repo
```

Comment:
Make sure to include [`cleanup_git_repo`](https://intel.github.io/dffml/plugins/dffml_operation.html#cleanup-git-repo)